### PR TITLE
ISSUE-2760 Resolving compatibility issues with Microsoft Bot Framework emulator

### PR DESCRIPTION
**Pull request by justinas-b**
**Proposed changes**:
- Updated BotFramework connector to address https://github.com/RasaHQ/rasa/issues/5056
- fixed https://github.com/RasaHQ/rasa/issues/5056

### CODE REVIEW

**Comment by erohmensing** [comment]
Thanks for the PR @justinas-b, it looks good! Can you please add a changelog entry via [these instructions](https://github.com/RasaHQ/rasa/tree/master/changelog)? The failing tests are something we're working on at the moment, and shouldn't be related to your changes.

**Comment by justinas-b** [comment]
Hi @erohmensing , I have updated CHANGELOG

File: changelog/5057.bugfix.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Addressing ISSUE-2760 to resolve compatibility issues with Microsoft Bot Framework Emulator.
```
Comment (erohmensing) [inline changelog/5057.bugfix.rst]:
```suggestion
Fixed compatibility issue with Microsoft Bot Framework Emulator if ``service_url`` lacked a trailing ``/``.
```

Comment (erohmensing) [reply changelog/5057.bugfix.rst]: @justinas-b can you please commit this change as I don't have push access to your fork?

**Comment by erohmensing** [comment]
Thanks for your contribution @justinas-b !

**Pull request merged**